lib/workload/stateless/stacks/sample-sheet-check/sample-sheet-check-lambda/src/samplesheet.py

```diff
@@ -108,6 +108,12 @@ def check_sample_name_format(self):
             logger.error("Sample ID {} did not have corresponding Sample_Name".format(self.sample_id))
             raise SampleNameFormatError
 
+        # Check if the Sample_Name column is the library_id from the Sample_ID column ignoring the "_topup"
```

Comment:
Mainly this, where it checks if the Sample_Name (which should be the library_id) is the same Sample_ID format (which is `{sampleID}_{library_ID}`)